Gems/LyShine/Code/Source/Animation/UiAnimationSystem.cpp

```diff
@@ -762,7 +765,7 @@ void UiAnimationSystem::ShowPlayedSequencesDebug()
                 names.push_back(name);
             }
 
-            //gEnv->pRenderer->Draw2dLabel((21.0f + 100.0f * i), ((i % 2) ? (y + 8.0f) : y), 1.0f, alreadyThere ? white : purple, false, "%s", name.c_str());
+Draw2dLabel((21.0f + 100.0f * i), ((i % 2) ? (y + 8.0f) : y), 1.0f, alreadyThere ? white : purple, false, "%s", name.c_str());
```

Comment:
I think the code should be removed or left commented if doesn't have a real implementation.

Also whitespace is missing